src/main/java/lotto/models/enums/Rank.java

```diff
@@ -0,0 +1,37 @@
+package lotto.models.enums;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum Rank {
+
+    FIRST(6,2000000000),
+    SECOND(5,1500000),
+    THIRD(4,50000),
+    FOURTH(3,5000),
+    OTHER(0, 0);
+
```

Comment:
```suggestion

    FIRST(6, 2_000_000_000),
    SECOND(5, 1_500_000),
    THIRD(4, 50_000),
    FOURTH(3, 5_000),
    OTHER(0, 0);

```
언더바로 세 자리씩 띄워주면 가독성을 높일 수 있겠네요.